aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -444,4 +451,15 @@ public boolean isS3ChecksumEnabled() {
   public void setS3ChecksumEnabled(boolean eTagCheckEnabled) {
     this.isS3ChecksumEnabled = eTagCheckEnabled;
   }
+
+  public Set<Tag> getS3WriteTags() {
+    return s3WriteTags;
+  }
+
+  public Set<Tag> toTags(Map<String, String> properties, String prefix) {
+    return PropertyUtil.propertiesWithPrefix(properties, prefix)
+        .entrySet().stream()
+        .map(e -> Tag.builder().key(e.getKey()).value(e.getValue()).build())
+        .collect(Collectors.toSet());
+  }
```

Comment:
[question] any reason why we made it public, as earlier it was private

Reply:
It can be private here (unlike other methods of `AwsProperties` which are public). I don't see a scope of `toTags` method to be used by other classes. We can make it public later on, if required.